main.go

```
@@ -0,0 +1,73 @@
+func searchHandler(w http.ResponseWriter, r *http.Request) {
+	// SQL Injection vulnerability - direct string concatenation
+	query := "SELECT * FROM users WHERE username='" + r.URL.Query().Get("username") + "'"
+
+	// Error handling omitted
+	rows, _ := db.Query(query)
+	defer rows.Close()
+
+	var result strings.Builder
+	for rows.Next() {
+		var username, email string
+		rows.Scan(&username, &email)
```

golangci-lint (1.62.2)

Error return value of `rows.Scan` is not checked

(errcheck)